### DISCUSSION TITLE
Consultation on using mkdocs material for freelancers directory project

### DISCUSSION THREAD

**Discussion by damascene** [squidfunk/mkdocs-material#6931] [Q&A]
Hi, First I really appreciate the effort that is put in this project and it's one the most amazing static pages creator I've seen. This encouraged me to try to use it for my freelancers directory project. Hopefully I'm not stretching it too much.

I'm using mkdocs material for displaying freelancer profiles that are based on a standard `toml` file format that is converted to `markdown`.

It work well for single freelancer, anyone can fork and create their own page.

## Example Profile
Here are an example profile using Mkdocs Material

<img src="https://github.com/squidfunk/mkdocs-material/assets/411046/670c8fa4-1341-456e-aead-15e7add8a1ae" width="300">
<img src="https://github.com/squidfunk/mkdocs-material/assets/411046/78c6778f-597d-4368-bb65-e3f5817cba84" width="300">

Link: https://hur-project.gitlab.io/mkdocs-hur-single-user/com.gitlab.example_user/

## Handling More Pages

However to be able to host multiple freelancer profiles I would like to have a way to show pages based on multiple tag selections, Search should be able to show all results on one page, and some other features. This could be based on data available from the original `toml` or the `markdown` file.

So I wanted to consult community before going further. 

## More Resources

If you want to understand the project better here are few links:

* Profile (`toml`) generating tool [demo](https://hur-project.gitlab.io/hur-frontend-sveltekit/) - [source code](https://gitlab.com/hur-project/hur-frontend-sveltekit)
* Mkdocs Material [Gitlab](https://gitlab.com/hur-project/mkdocs-hur-single-user) and [Github](https://github.com/damascene/hur-mkdocs-single-freelancer) repository for freelancers to fork - [demo](https://hur-project.gitlab.io/mkdocs-hur-single-user/com.gitlab.example_user/)

Project aims to make it easier for freelancers and customers to communicate and to provide an open database that could be used by different software tools for display, indexing and filtering.

**Comment by alexvoss** [comment]
Thanks for sharing. There is quite a lot of stuff in the Insiders Edition that I think would be useful here. I am thinking of [nested tags](https://squidfunk.github.io/mkdocs-material/setup/setting-up-tags/#nested-tags) in particular. Think `#programming/C++` or `#delivery/jenkins`. There are also [multiple index pages](https://squidfunk.github.io/mkdocs-material/plugins/tags/#listing-configuration) that can be scoped to a part of the site. You can set up [allowed tags](https://squidfunk.github.io/mkdocs-material/plugins/tags/#config.tags_allowed) in the configuration if desired. Your projects sounds like a cool use case for these features.

The [new search](https://github.com/squidfunk/mkdocs-material/issues/6307) should help with searching for freelancers in the directory. It will allow for faceted searching, for example, though that will likely be an Insiders feature initially.

Note that all these features will ultimately make it into the public version, though we need to hit a few funding goals on the way. Am optimistic that we will!

**Reply by damascene** [reply]
Thank you for your comments I'll check those link d.v.  I'm happy that you see MKdocs Material will satisfy my need with few tweaks. I hope more community members will give suggestions and recommendations.
